src/components/product/dto/create-product.dto.ts

```diff
@@ -51,6 +51,15 @@ export abstract class CreateProduct {
   @Type(() => ScriptureRangeInput)
   readonly scriptureReferencesOverride?: ScriptureRangeInput[];
 
+  @Field({
+    nullable: true,
+    description: stripIndent`
+      When isOverriding is false, scriptureReferencesOverride will be ignored.
+      Note only \`DerivativeScriptureProduct\`s can use this field.
+    `,
+  })
+  readonly isOverriding?: boolean = false;
```

Comment:
This doesn't need to be apart of the API schema. It's only for the DB to help give correct response to the API schema already in place.